### ISSUE TITLE
See different edit/import/export options for upgraded metadata?

### ISSUE THREAD

**Issue by jlholder** [Innovate-Inc/EMEMetadataToolKit#154]
Going through the same metadata upgrade process, after metadata is in ArcGIS style, I get different options. Does that mean something in the upgrade isn't quite successful? 

Sometimes I see these options:
![image](https://cloud.githubusercontent.com/assets/17882165/26066849/4f1430c0-3966-11e7-87e2-11d947cb63d0.png)

sometimes I see these:
![image](https://cloud.githubusercontent.com/assets/17882165/26066926/836deb0e-3966-11e7-90d7-5ce71f46f908.png)

and sometimes these:
![image](https://cloud.githubusercontent.com/assets/17882165/26067115/122062c8-3967-11e7-9046-7b7b26260610.png)

**Comment by torrin47** [comment]
Yes, you wouldn't see the Upgrade button if the upgrade was successful. And the first instance would seem to indicate that ArcCatalog doesn't recognize the format.

**Comment by jlholder** [comment]
So I have the original metadata. I've tried upgrading using the EME Toolbox ("EPA Upgrade FGDC to ArcGIS) and the Upgrade button in the description above, both ESRI ISO to ArcGIS and FGDC to ArcGIS options just for giggles. I'm not sure what other options I have? I still don't get to the final print/edit/validate/export/import settings.

The record says its FGDC: https://edg.epa.gov/metadata/rest/document?id=%7BC27775F4-DDB4-40E0-B7F0-83827D55AED6%7D&xsl=metadata_to_html_full

Any suggestions of where to look or other upgrade options? What other similar format would we have stored EDG metadata in from 2004? Thanks!
